tuxemon/core/components/menu/save_menu.py

```diff
@@ -40,12 +40,12 @@ def draw(self):
             slot_name = "slot" + str(slot_num)
 
             # Check to see if a save exists for the current slot
-            if os.path.exists("saves/slot" + str(slot_num) + ".png"):
+            if os.path.exists(".saves/slot" + str(slot_num) + ".png"):
```

Comment:
Makes sense, this was part of my questions I had when I was asking about the folder in the first place.